i18n/fr.yaml

```
@@ -1,33 +1,34 @@
-join_slack: "Join Slack"
-more_videos: "More Videos"
+join_slack: "Rejoindre Slack"
+more_videos: "Plus de Vidéos"
 
 # Mailing List
-join_mailing_list: "Join Mailing List"
-enter_email: "Enter Email"
-subscribe: "Subscribe"
+join_mailing_list: "Rejoindre la Liste de Diffusion"
+enter_email: "Entrez votre Email"
+subscribe: "Souscrire"
 
 # About
 about_innersource_commons: |-
-  InnerSource Commons is the world's largest community of InnerSource practitioners. It is dedicated to creating and sharing knowledge about InnerSource, the use of open source best practices for software development within the confines of an organization.
+  InnerSource Commons est la plus grande communauté au monde de praticiens de l'InnerSource. Elle est dédiée à la création et au partage de connaissances sur l'InnerSource, l'utilisation des bonnes pratiques de l'Open Source pour le développement de logiciel à l'intérieur d'une organisation.
```

```suggestion
  InnerSource Commons est la plus grande communauté au monde de praticiens de l'InnerSource. Elle est dédiée à la création et au partage de connaissances sur l'InnerSource, l'utilisation des bonnes pratiques de l'Open Source pour le développement de logiciels à l'intérieur d'une organisation.
```
(or "développement logiciel")

---

taken into account